sympy/sets/sets.py

```diff
@@ -1130,6 +1140,16 @@ def as_relational(self, symbol):
                 return And(Ne(symbol, a.sup), symbol < b.sup, symbol > a.inf)
         return Or(*[set.as_relational(symbol) for set in self.args])
 
+    @property
+    def is_empty(self):
+        a, b = self.args
+        if a.sup >= a.inf or b.sup >= b.inf:
```

Comment:
I think the logic could be something like this:

    if all arguments are empty:
        return True
    if any argument is not empty:
        return False
    else:
        return None